cmd/logs.go

```diff
@@ -14,17 +14,33 @@ const (
 	warnLevelText  = "Warning: "
 )
 
+func init() {
+	// TODO: uncomment when pack runs lifecycle with a tty
```

Comment:
Is there an issue associated with this work? It would be nice to have the issue link here.

Reply:
https://github.com/buildpacks/pack/issues/493 which appears no closer than ever to being worked.  And as predicted, the longer we wait to implement it, the more people are just forcing TTY-driven behaviors to enabled, causing a sub-par experience for systems that aren't TTY (like `kpack` and log warehousing).